src/test/resources/fixtures/hmh/hmh-insighted/development/caliperGradeEventScore.json

```diff
@@ -0,0 +1,72 @@
+{
+  "sensor": "https://www.hmhco.com/AggPA",
+  "sendTime": "2019-03-22T17:46:40.111Z",
+  "dataVersion": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "data": [
+    {
+      "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+      "type": "GradeEvent",
+      "id": "urn:uuid:dc790a2d-124e-4b20-9f89-2bca87ea22d4",
+      "actor": {
+        "id": "urn:uuid:d76d4cbe-0f12-4323-af07-e68566dcbcd4",
+        "type": "Person"
+      },
+      "action": "Graded",
+      "object": {
+        "id": "urn:uuid:830b77b2-f4b3-4ac3-87b0-a4fef1d7c33e",
+        "type": "Attempt",
+        "assignable": {
+          "id": "urn:uuid:376835ba-947a-47f3-a990-003c7eb11cfc",
+          "type": "Assessment",
+          "extensions": {
+            "disciplineCode": "ED18_RLA"
+          },
+          "creators": [
```

Comment:
this field isnt not currently supported by Ed Analytics - will flag it to the team